crypto/x509/rsa_pss.c

```diff
@@ -229,7 +229,7 @@ int x509_rsa_ctx_to_pss(EVP_MD_CTX *ctx, X509_ALGOR *algor) {
 
   if (saltlen != 20) {
```

Comment:
I couldn't spot in the history of this file where we diverted from upstream. Why we don't have the assertion and comment as in there?

Reply:
https://github.com/google/boringssl/blob/master/crypto/x509/rsa_pss.c#L121
i.e. there is no allowed digest algorithm with output length of 20 bytes. Not the case for us.

Reply:
Thanks, Torben. I see we tracked skipping upstream's commit [Tighten up supported PSS combinations in X.509.](https://github.com/google/boringssl/commit/a6d321b11fa80496b7c8ae6405468c212d4f5c87) in CryptoAlg-1331.